integration/airflow/tests/integration/requests/snowflake.json

```diff
@@ -13,7 +13,7 @@
 	"outputs": [{
 		"facets": {
 			"dataSource": {
-				"name": "snowflake://{{ env_var('SNOWFLAKE_ACCOUNT_ID') }}",
+				"name": "snowflake://gp21411.us-east-1.aws",
```

Comment:
That would be great if we could restore templated expected result.
I have a solution why it fails: `urlparse` used in `fix_snowflake_sqlalchemy_uri` returns `hostname` which is always lowercase according to docs. Adding ` | lower` filter to Jinja template should resolve the issue.